src/results/result-form-field.component.tsx

```diff
@@ -23,6 +24,28 @@ const ResultFormField: React.FC<ResultFormFieldProps> = ({
   const isCoded = (concept) => concept.datatype?.display === "Coded";
   const isPanel = (concept) => concept.setMembers?.length > 0;
 
+  const printValueRange = (concept: ConceptReference) => {
+    let maxVal;
+    let minVal;
+    let rangeString = "";
+    if (concept?.datatype?.display === "Numeric") {
+      maxVal = Math.max(
+        concept?.hiAbsolute,
+        concept?.hiCritical,
+        concept?.hiNormal
+      );
+      minVal = Math.min(
+        concept?.lowAbsolute,
+        concept?.lowCritical,
+        concept?.lowNormal
+      );
+      rangeString = ` (${minVal ?? 0} - ${maxVal ?? 0} ${
+        concept?.units ?? ""
+      })`;
+    }
+    return rangeString;
```

Comment:
can we make the ranges or the Test bold atleast

Reply:
How is that done? inline styling?